ch04.adoc

```diff
@@ -185,8 +185,12 @@ The `computed_standard_name` attribute indicates that the values in variable
 [[time-coordinate, Section 4.4, "Time Coordinate"]]
 === Time Coordinate
 
-Variables representing reference time must always explicitly include the **`units`** attribute; there is no default value.
-The **`units`** attribute takes a string value that follows the formatting requirements of the <<UDUNITS>> package. These requirements can best be described by an example with explanatory comments:
+A time coordinate identifies an instant along the continuous physical dimension of time, whether in reality or a model.
+Variables containing time coordinates must always explicitly include the **`units`** attribute.
+The **`units`** attribute takes a string value that follows the formatting requirements of the <<UDUNITS>> package.
+It must comprise a unit of measure that is physically equivalent to the SI base unit of time (i.e. the second), followed by the word **`since`** and a reference date-time. 
+There is no default value for the **`units`**.
+These requirements can best be described by an example with explanatory comments:
 
 The time unit specification **`seconds since 1992-10-8 15:15:42.5 -6:00`** indicates seconds since October 8th, 1992  at  3 hours,  15 minutes  and  42.5 seconds in the afternoon in the time zone which is six hours to the west of Coordinated Universal Time (i.e.  Mountain Daylight Time).
```

Comment:
Maybe this should. be "**e.g.** Mountain Daylight Time" as it can  also other time / DST  zones.